proxy/coco/src/request/existential.rs

```diff
@@ -0,0 +1,177 @@
+// I reserve the right to not match all the arms when picking out particular cases, thank you very
```

Comment:
Why is this module called "existential"?

Reply:
The request (temporarily) forgets what state it's in and becomes `SomeRequest`

Reply:
> **Existentialism** (/ˌɛɡzɪˈstɛnʃəlɪzəm/ or /ˌɛksəˈstɛntʃəˌlɪzəm/) is a form of philosophical enquiry that explores the nature of existence by emphasizing experience of the human subject—not merely the thinking subject, but the acting, feeling, living human individual.